pyaqueduct/client/client.py

```diff
@@ -480,13 +471,85 @@ def execute_extension_action(
                 error.errors if error.errors else "Unknown error occurred in the remote operation."
             ) from error
 
-        result = ExtensionExecutionResultData.from_dict(
-            extension_result["executeExtension"]  # pylint: disable=unsubscriptable-object
-        )
+        result = ExtensionExecutionResultData.from_dict(extension_result["executeExtension"]) # pylint: disable=unsubscriptable-object
```

Comment:
This and similar changes (1) remove double space before comment (2) not related to a feature. May you please bring them back?